spec/classes/failover_spec.rb

```diff
@@ -15,11 +15,12 @@
           :peer_address => '10.1.1.20',
         } end
 
-        let(:facts) do {
-          :concat_basedir => '/doesnotexist',
-          :domain         => 'example.org',
-          :osfamily       => osfamily,
-        } end
+        let(:facts) do
+          facts.merge({
+            :concat_basedir => '/doesnotexist',
+            :domain         => 'example.org',
```

Comment:
Fully agree with that. I kept it, because I thought that the "fragment content check" also checks the domain name, but this is only the case in `init_spec.rb`. I will correct this.